### PR TITLE
Whole Swathe of Translations 2

### PR DESCRIPTION
**Pull request by KomradeSpectre**
Completely translated Augments/Augment Names (Also corrected a few wrongslations.)
Mostly translated p0003.xml .
Mostly translated Caplink stuff (why? I'm not sure myself.)
Only a few Item Info translations.
Fixed a misnomer (Demon Unrefined Wine to Unrefined Demon Wine)
Fixed a misnomer I made (Hidell Catacombs referenced Hidell Church instead of Hidell Chapel)
Fixed some misnomers in normal skill stuff (Magic to Magick, Hack to Carve on the Seeker, etc)
Translated some Bazaar stuff.

Format changed in accordance to desired changes in last PR.
If I missed anything in regards to this, please let me know. I'm gonna take a small
break from translating (a day or so) before completely translating skills haha

### CODE REVIEW

**Comment by KomradeSpectre** [comment]
Weird that it's tracking old commits on here.
On that note, I will be fully translating all skills from all classes, so expect those to appear below.

**Comment by KomradeSpectre** [comment]
Figured that &lt;KEYM&gt; and &lt;PADM&gt; must be cases where it displays the specific section (keyboard based or gamepad based) when these tags lead the section. Fixed the spirit lancer stuff accordingly. If I'm wrong on this, let me know. Markdown is altering the things I mean, keeping them from displaying here, you can check what I mean in the commit.

File: ui/00_message/skill/custom_skill_info_01.xml [hunk 1]
```diff
@@ -136,6 +136,12 @@ LV.6: Increases range. Increases counter duration.&lt;/COL&gt;</translation>
 初段を敵に接触させれば連撃に移行する
 &lt;COL ffdc78&gt;LV UP毎に攻撃威力の上昇
 LV.６でモーション変化／攻撃ヒット数増加&lt;/COL&gt;</original>
+    <translation>A high-powered sword dance that covers a wide area.
+If the first hit is successfully landed, it will become a full on
+combo attack.
+&lt;COL ffdc78&gt;LV UP: Increases damage of the skill.
+LV.６: Increases the number of hits in the attack, and changes
```
Comment (riftcrystal) [inline ui/00_message/skill/custom_skill_info_01.xml]: Please change the fullwidth character to the regular character (in this case, LV.６should be LV.6). This applies to any other translated messages as well.

File: ui/00_message/common/item_name.xml [hunk 2]
```diff
@@ -19659,7 +19659,7 @@
   </item>
   <item id="7839">
     <original>鬼どぶろく</original>
-    <translation>Demon Unrefined Wine</translation>
+    <translation>Unrefined Demon Wine</translation>
```
Comment (riftcrystal) [inline ui/00_message/common/item_name.xml:19662]: You're certain this is an error? I try to match the DDON translation wikidot for easier end-user use.

File: ui/00_message/skill/ability_info.xml [hunk 3]
```diff
@@ -2264,6 +2332,9 @@ LV5～6: Increases stagger power.&lt;/COL&gt;</translation>
     <original>スラストのダメージ量が増加する
 &lt;COL ffdc78&gt;LV UP毎にダメージがより増加
 LV5～6で吹き飛ばし力が増加&lt;/COL&gt;</original>
+    <translation>Increases damage of Thrust.
+&lt;COL ffdc78&gt;LV UP: Increases damage further.
+LV5～6: Increases blow power.&lt;/COL&gt;</translation>
```
Comment (riftcrystal) [inline ui/00_message/skill/ability_info.xml]: Please change the fullwidth character to the regular character (in this case, ～ should be ~). This applies to any other translated messages as well.

File: ui/00_message/skill/ability_info.xml [hunk 4]
```diff
@@ -2696,61 +2831,92 @@ LV5～6で吹き飛ばし力が増加&lt;/COL&gt;</original>
...
+LV5～6: Increases chance and exhaust attack.&lt;/COL&gt;</translation>
   </message>
   <message>
     <key>ABILITY_INFO_326</key>
     <original>ヒールオーラ/ヒールオーラエクス/
 ライドヒールオーラの回復量が増加する
 &lt;COL ffdc78&gt;LV UP毎に回復量がより増加
 LV5～6で展開中の耐久力が増加&lt;/COL&gt;</original>
+    <translation>Increases the recovery amount of
+Heal Aura / Greater Heal Aura / Ride Heal Aura.
```
Comment (riftcrystal) [inline ui/00_message/skill/ability_info.xml:2854]: I don't think the spaces around the / between skill names is necessary here.

File: ui/00_message/skill/normal_skill_name_06.xml [hunk 5]
```diff
@@ -2,17 +2,17 @@
   <message>
     <key>NORMAL_SKILL_NAME_06_92</key>
     <original>マジックボール</original>
-    <translation>Magic Balls</translation>
+    <translation>Magick Bolts</translation>
```
Comment (riftcrystal) [inline ui/00_message/skill/normal_skill_name_06.xml:5]: Why balls => bolts?

File: ui/00_message/skill/normal_skill_name_08.xml [hunk 6]
```diff
@@ -2,6 +2,7 @@
   <message>
     <key>NORMAL_SKILL_NAME_08_113</key>
     <original>&lt;PADM&gt;闘波の構え&lt;KEYM&gt;構えの変更（闘波の構え）</original>
+    <translation>&lt;PADM&gt;Seeker Aim&lt;KEYM&gt;Change Stance(Seeker Aim)</translation>
```
Comment (riftcrystal) [inline ui/00_message/skill/normal_skill_name_08.xml]: Need a space before the parenthesis for Change Stance.

File: ui/00_message/ui/Caplink_res.xml [hunk 7]
```diff
@@ -40,42 +41,52 @@
   <message>
     <key>caplink_dialog_is_captalk</key>
     <original>%sとカプトークを開始しますか？</original>
+    <translation>Would you like to start Captalk with %s?</translation>
   </message>
   <message>
     <key>caplink_dialog_done_invite_game</key>
     <original>%sを招待しました。</original>
+    <translation>Invited %s .</translation>
```
Comment (riftcrystal) [inline ui/00_message/ui/Caplink_res.xml]: Errant space here before the period.

File: ui/00_message/ui/Caplink_res.xml [hunk 8]
```diff
@@ -40,42 +41,52 @@
...
   </message>
   <message>
     <key>caplink_button_check</key>
     <original>確認</original>
+    <translation>Confirm</translation>
   </message>
   <message>
     <key>caplink_dialog_done_offer_captomo</key>
     <original>%sにカプトモ申請をしました。</original>
+    <translation>Sent a Capfriend request to %s .</translation>
```
Comment (riftcrystal) [inline ui/00_message/ui/Caplink_res.xml]: Errant space here before the period.

**Review by riftcrystal — CHANGES_REQUESTED** [review]

File: ui/00_message/skill/normal_skill_name_06.xml (hunk 5 above)
Comment (KomradeSpectre) [reply ui/00_message/skill/normal_skill_name_06.xml:5]: It's a skill mirror from the other game. Bolts make sense if we have skill mirror names from DD:DA in here.

File: ui/00_message/common/item_name.xml (hunk 2 above)
Comment (KomradeSpectre) [reply ui/00_message/common/item_name.xml:19662]: I'm certain that's a grammatical error. If that's how the White Dragon Temple translated it, they mistranslated it.

Comment (KomradeSpectre) [reply ui/00_message/common/item_name.xml:19662]: Not that it's the first mistranslation I've seen, but I don't really have all the time to go through and catch all of them haha

File: ui/00_message/skill/normal_skill_name_06.xml (hunk 5 above)
Comment (KomradeSpectre) [reply ui/00_message/skill/normal_skill_name_06.xml:5]: For example, Devastate being in the warrior's skills despite the text of the skill not meaning Devastate.

File: ui/00_message/common/item_name.xml (hunk 2 above)
Comment (DaedalusGame) [reply ui/00_message/common/item_name.xml:19662]: From what I gather, it's referring to unfiltered/unrefined demon sake
The literal translation is oni doburoku; i don't think i need to explain oni here.
Doburoku is an unrefined sake that can easily be made at home from steamed rice, kōji, water and yeast. Creation of this sake was outlawed during the meiji period apparently.

Apart from this, grammatically, i think both of these are correct but Unrefined Demon Wine sounds better. It's unrefined "wine"(sake) made by demons.

**Review by riftcrystal — APPROVED** [review]

**Pull request merged**